core/sqf/conf/log4cxx.trafodion.masterexe.config

```diff
@@ -44,15 +45,15 @@ log4j.appender.mxoAppender.layout=org.apache.log4j.PatternLayout
 log4j.appender.mxoAppender.layout.ConversionPattern=%d, %p, %c, %m%n
 
 #MXOSRVR
+log4j.additive.mxoAppender=false 
 log4j.logger.MXOSRVR=ERROR, mxoAppender
 
 # SQL
 # uncomment the following line if you want to enable debug logging
-#log4j.logger.SQL.EXE=DEBUG
 log4j.logger.SQL=INFO
 log4j.logger.SQL.COMP=ERROR
-log4j.logger.SQL.EXE.Memory=INFO
-log4j.logger.SQL.ESP=ERROR
+log4j.logger.SQL.EXE=INFO
+log4j.logger.SQL.ESP=INFO
```

Comment:
I'm not familiar with this code path. But generally we want to avoid flood of INFO messages. It makes debugging difficult when we have to scan through the logs for real problems.